### DISCUSSION TITLE
Adding the CF Attributes to the NVS

### DISCUSSION THREAD

**Discussion by feggleton** [cf-convention/discuss#347] [Announcements relevant to CF]
Hi everyone, I spoke to a couple of people at the last CF annual workshop about adding the CF Attribute Names to the NERC Vocab Server and I had good support to do so.

A new NVS collection has been created here: http://vocab.nerc.ac.uk/collection/P31/current/

I will initially be adding the 52 attribute terms from the CF Conventions document, Appendix A, table of Attributes. This controlled vocabulary is set up very similar to the standard names collection. I will maintain this collection should new attributes be added. Any comments or questions, please add below.

Thanks, Fran

**Comment by larsbarring** [comment]
Would it be relevant to provide this link in Appendix A.

**Comment by feggleton** [comment]
Possibly.. do we link to the NERC Vocab Server when we discuss Standard Names in the conventions document?

**Comment by larsbarring** [comment]
I had the same thought as you; if we add this link to Appendix A, we should do the same for standard names which would be http://vocab.nerc.ac.uk/collection/P07/current/. And I think the right place to do this would be in the standard name table (which is what actually is in NVS) as well as in Section 3.3. 

Or, we do not add links in the Conventions document, but in the web site [Vocabularies](https://cfconventions.org/vocabularies.html) page, where a new section would then be needed for the Attributes.

**Comment by larsbarring** [comment]
.... or maybe we should not given the discussion in cf-convention/vocabularies#11 (which I just rigth now came across) ....

**Comment by davidhassell** [comment]
Hi, I think that making these connections on the website is better, and a good connection to make. The fact that these attributes are duplicated elsewhere doesn't really feel like part of the actual conventions themselves.

**Comment by rob-metalinkage** [comment]
Any progress on a resolution to this?  There is a STAC JSON schema - https://github.com/stac-extensions/cf

We can connect this to an ontology using JSON-LD - if we had one :-)

This is part of a GeoDCAT requirements and development process where we can build profiles to match community standards:

https://ogcincubator.github.io/geodcat-ogcapi-records/

early days - but open to collaboration.   Communities can engage to test, refine and promote these resources into formal standards as part of the OGC and ISO processes if appropriate.

**Comment by feggleton** [comment]
Coming back to this - https://github.com/cf-convention/cf-convention.github.io/blob/add-nvs-link/vocabularies.md -I have added the NVS links to the vocab website page on a branch to test this out before adding in - I've added the standard name link, the area type table and the standardised region list. I have then also added the CF Attributes in a new section at the bottom. I think this would be a good temporary solution until we potentially move to using the NVS as the primary source of the vocabulary which has been discussed elsewhere. 

This made me realise there are other CF related terms published in the NVS such as; CF calendars, CF Cell methods and CF Vertical Co-ordinate Coverages. Should these also feature on the website? I have had some interest from others recently about using the CF cell methods in other areas as they are a good set of standardised statistical methods. However it looks like this controlled vocabulary and others in the NVS are not up to date as they are not supported by an update process. This is something I'd like to discuss. 

The ontology creation question is in a separate discussion so we should discuss that there.
